rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -222,13 +231,13 @@ def try_all_rules(
             decoded_events: list['EvmEvent'],
             action_items: list[ActionItem],
             all_logs: list[EvmTxReceiptLog],
-    ) -> tuple[Optional['EvmEvent'], list[ActionItem]]:
+    ) -> DecodingOutput:
```

Comment:
Why not:

1. Add a short docstring here
2. Instead of returning empty decoding output just make it optional and return None?